source/resources/js/app/tools/strength.js

```diff
@@ -0,0 +1,26 @@
+"use strict";
```

Comment:
I think it would be a good idea to merge from master, since this file has already been added in the other PR.